### PR TITLE
Adding Application Docs

### PR DESCRIPTION
**Pull request by Xuan-Lim**
As referred to here (#34 ) in the issue, I suggested writing a documentation guiding the application process. I had some time so here you go:) 

Let me know if there are any changes that I should make, some advice I wouldn't have known having taken a different route, typos, anything helpful:)

### CODE REVIEW

File: applying_to_uol/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  * [English Requirements (for Non-English Speaking Countries)](#english-requirements-for-non-english-speaking-countries)
+  * [Documents to Prepare for Application](#documents-and-statement-to-prepare-for-application)
+* [Application Process](#application-process)
+* [Recognition of Prior Learning (RPL) Application](#recognition-of-prior-learning-rpl-application)
+* [Frequently Asked Questions (FAQs)](#faqs)
+* [Additional Resources](#additional-resources)
+
+## Administrative Information: Admissions Process and Requirements
+
+Students may apply to the programme in two of the following ways: Direct Admissions (where you are admitted to study as conventionally expecte) and Performance-based Admissions (often used as an alternative route to getting admitted if you did not meet the academic requirements). Regardless of which route you would be falling under, you must meet an equivalent of three requirements for your track:
```
Comment (sglavoie) [inline applying_to_uol/README.md]: Typo: expecte → expected

File: applying_to_uol/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+1. The entry requirements for your track of admissions
+2. BSc Comp Sci's specific requirements of Maths at the appropreate level
```
Comment (sglavoie) [inline applying_to_uol/README.md]: Typo: appropreate → appropriate

File: applying_to_uol/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+3. [English Requirements](#english-requirements-for-non-english-speaking-countries)
+
+For Direct Admissions: General Requirements must be met - your country's equivalent can be found here: https://london.ac.uk/entrance-qualifications
+
+For Performance-Based Admissions: your requirements can be found here: https://www.coursera.org/degrees/bachelor-of-science-computer-science-london/admissions
+
+If you're not sure which track you'll fall under - or whether or not you will be able to obtain admissions, UOL's general advice is to apply early and to apply anyway (see [Application Process](#application-process))
```
Comment (sglavoie) [inline applying_to_uol/README.md]: For consistency: UOL → UoL.

File: applying_to_uol/README.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+### Documents and Statement to Prepare for Application
+
+If you're familiar with the British UCAS system, which students use
+after sixth form (12/13th Grade) apply to university, the application is fairly similar. The only difference being that you're applying to one university and not five.
+
+In the application you will be asked to supply them with the following things. I would suggest having these prepared beforehand (especially academic documents) to ensure that your application decisions come in as smoothly as possible:
+
+1. A scanned copy of your ID
+2. A verfied copy of the Academic Documents that back your meeting the requirements for the programme
```
Comment (sglavoie) [inline applying_to_uol/README.md]: Typo: verfied → verified.

File: applying_to_uol/README.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+2. [Discretionary RPL: You have a diploma/incomplete degree you would like the programme director to consider for RPL](https://london.ac.uk/applications/how-apply/recognition-prior-learning/recognition-and-accreditation-prior-learning-3)
+3. [Google's IT Diploma: You completed the course and would like the HCW exemption](https://www.coursera.org/professional-certificates/google-it-support)
+
+RPL from a discretionary route will take awhile, students have reported the process taking a full month before the process was complete. It would be wise to apply as soon as possible to avoid delays with module registration.
+
+## FAQs
+
+**Q:** My application to UoL/for RPL/some other administrative process is taking so long, have they received it/why's it taking so long?
+
+**A:** It takes forever. Sorry mate. That's why it's important to apply early for peace of mind. It means that your application isn't clogged up and done when professors aren't stressed out when they read your application.
```
Comment (sglavoie) [inline applying_to_uol/README.md]: Two extra spaces at the end.

File: applying_to_uol/README.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
+# Applying to the University of London Bachelors of Computer Science degree (UoL)
```
Comment (sglavoie) [inline applying_to_uol/README.md]: Two extra spaces at the end.

File: applying_to_uol/README.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+**A:** It looks like [this](https://www.thestudentroom.co.uk/showthread.php?t=2459201).
+
+**Q:** Where would we be graduating?
+
+**A:** [Here, at The Barbican in London](https://www.youtube.com/watch?v=Oja7n2Kq2do)
+
+## Additional Resources
+
+[UoL Unofficial Students' Discord](https://discord.gg/fugMPVR)
```
Comment (sglavoie) [inline applying_to_uol/README.md]: Preferably prefix with hyphen for consistency:
- [UoL Unofficial Students' Discord](https://discord.gg/fugMPVR)

File: applying_to_uol/README.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
```
Comment (sglavoie) [inline applying_to_uol/README.md]: This line can be removed once hyphens are added.

File: applying_to_uol/README.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+[UoL Facebook](https://www.facebook.com/LondonU/?ref=br_rs)
```
Comment (sglavoie) [inline applying_to_uol/README.md]: Preferably prefix with hyphen for consistency:
- [UoL Facebook](https://www.facebook.com/LondonU/?ref=br_rs)

**Comment by Xuan-Lim** [comment]
There you go! Thanks for reviewing my code with me. Reminded me to set my VSCode configs and addons to what they should be:)

**Comment by sglavoie** [comment]
Done! Thank you very much for sharing! I hope you'll find your experience at UoL enriching and that you'll get out of it exactly what you were looking for. Looking forward to seeing you on Slack at the very least :wink:.

**Comment by sglavoie** [comment]
@all-contributors please add @Xuan-Lim for doc

**Pull request merged**